packages/connection-form/src/components/connect-form-actions.tsx

```diff
@@ -8,18 +8,28 @@ import {
   palette,
   css,
   cx,
+  useDarkMode,
```

Comment:
are changes to this file related to the current PR?

Reply:
no. commit message "drive-by". I saw that line too many times :)

Reply:
I can extract it, I suppose

Reply:
was just curious. you can leave it!